docs/entities/blocks/block.schema.json

```diff
@@ -3,7 +3,7 @@
   "title": "Block",
   "description": "A block",
   "type": "object",
-  "required": ["canonical", "height", "hash", "parent_block_hash", "txs", "burn_block_time", "burn_block_time_iso", "burn_block_hash", "burn_block_height", "miner_txid"],
+  "required": ["canonical", "height", "hash", "parent_block_hash", "txs", "burn_block_time", "burn_block_time_iso", "burn_block_hash", "burn_block_height", "miner_txid", "parent_microblock_hash", "parent_microblock_sequence", "microblocks_accepted", "microblocks_streamed"],
```

Comment:
Could we run prettier on the schema json too? Think this will auto-format this on new lines.

Reply:
Yeah we should. I also have a related couple TODOs in my notes:
* Print a warning in the doc type gen for schema objects that are missing a `title`.
* Print a warning in the doc type gen for schema objects that are missing `additionalProperties: false`

Reply:
Issue created https://github.com/blockstack/stacks-blockchain-api/issues/657